example/src/HeadlessScreen.tsx

```
@@ -21,6 +21,7 @@ export default function HeadlessPage() {
     const APP_ID = "YOUR_APP_ID";
 
     useEffect(() => {
+
```

ESLint

[error] Replace `····useEffect(()·=>·{⏎` with `··useEffect(()·=>·{`

(prettier/prettier)